app/client/src/components/shared/AboutContent.js

```diff
@@ -407,6 +408,23 @@ function AboutContent() {
                 </p>
               </div>
             </TabPanel>
+            <TabPanel>
+              <div className="container">
+                <h1>Educational Materials from How's My Waterway</h1>
+                <hr />
+                <p>
+                  <ul>
+                    <li>
+                      <em>How's My Waterway</em> middleschool lesson plan:{' '}
+                      <br />
+                      <a href="https://www.epa.gov/waterdata/hows-my-waterway-middle-school-lesson-plan">
+                        https://www.epa.gov/waterdata/hows-my-waterway-middle-school-lesson-plan
+                      </a>
```

Comment:
No need to duplicate content – especially since this link is set in the `EducatorsContent` from the configurable JSON file. I'll replace all this duplicated education content defined here with what's defined in the `EducatorsContent` component (where you can, embrace composition over duplication).